easycv/predictors/detector.py

```diff
@@ -36,6 +35,184 @@
     from easycv.thirdparty.mtcnn import FaceDetector
 
 
+@PREDICTORS.register_module()
+class DetectionPredictor(PredictorInterface):
+
+    def __init__(self, model_path, config_path=None):
+
+        self.model_path = model_path
+
+        if config_path is not None:
+            self.cfg = mmcv_config_fromfile(config_path)
+        else:
+            if is_url_path(self.model_path) and url_path_exists(
```

Comment:
how to determine whether the model_path is a checkpoint path or an exported model path?

Reply:
universal. config_path is to avoid inconsistent model_path versions.